src/App/App.tsx

```diff
@@ -112,7 +124,7 @@ export default class App extends Component<any, any> {
     this.setState(prevState => ({
       // @ts-ignore
       showLoginModal: !prevState.showLoginModal,
-      error: {},
```

Comment:
Do you remember which rule was yelling here? I do not remember perfectly whether the literal object is required for something else.

Reply:
My bad error should be removed here instead of setting to undefined since `error` should be optional?
error is changed to an optional type because Typescript throws error when using spread params and it makes more sense if no errors to not set it as an empty object which would change its signature